pkg/model/crd.go

```diff
@@ -618,6 +618,32 @@ func (r *CRD) ListOpMatchFieldNames() []string {
 	return r.cfg.ListOpMatchFieldNames(r.Names.Original)
 }
 
+// GetAllRenames returns all the field renames observed in the generator.yaml.
+func (r *CRD) GetAllRenames() (map[string]string, error) {
+	resourceConfig, ok := r.cfg.Resources[r.Names.Original]
+	if !ok {
+		return nil, fmt.Errorf("resource not found")
+	}
+
+	renames := make(map[string]string)
+	opMap := r.sdkAPI.GetOperationMap(r.cfg)
+	createOps := (*opMap)[OpTypeCreate]
```

Comment:
Is this necessary? I understand it's future proofing, but either we should iterate through all operations or only iterate through create.

Reply:
We should be only interested by `CreateOp` renames when using this function. But that's a good point as well - i believe we should pass an op parameter to the function